.travis.yml

```diff
@@ -5,11 +5,13 @@ git:
   depth: 9999999
 jdk:
 - oraclejdk8
+- openjdk8
```

Comment:
Can you explain whether this will cause us to run the test suite twice, once with Oracle JDK and once with OpenJDK? Will it affect our test suite runtime at all?